Inverted-Index/src/FileReader.java

```diff
@@ -13,22 +13,26 @@
         File filesList[] = directoryPath.listFiles();
         List<Doc> docs = new ArrayList<>();
   
-        Scanner sc = null;
         for(File file : filesList) {
+            docs.add(fileToDoc(file));
```

Comment:
You could make a constructor for `Doc` that accepts a file instead of implementing  `fileToDoc` method.